.github/pull_request_template.md

```diff
@@ -1,10 +1,10 @@
-# Related Issue/Addition to code
+### Related Issue/Addition to code
 <!-- Please delete options that are not relevant. -->
 
 - Fixes #
 - Issue/Addition to code, goes here.
```

Comment:
Is the goes here, really needed? Won't Issue/Addition to code be enough too? What do you think.